mempool/reactor.go

```diff
@@ -224,11 +224,16 @@ func (r *Reactor) processMempoolCh() {
 		case envelope := <-r.mempoolCh.In():
 			if err := r.handleMessage(r.mempoolCh.ID(), envelope); err != nil {
 				r.Logger.Error("failed to process message", "ch_id", r.mempoolCh.ID(), "envelope", envelope, "err", err)
+
+				fmt.Println("MESSAGE HANDLER ERROR:", err)
```

Comment:
???

Reply:
It was removed ;-)

Reply:
This was my bad -- I shouldn't have added debug logic w/ automerge enabled.